pydantic/function_model.py

```diff
@@ -0,0 +1,442 @@
+"""Provides FunctionModel, a model for validating function arguments."""
+
+from collections import OrderedDict
```

Comment:
IIRC all Python dicts are guaranteed to be ordered since Python 3.6 and `pydantic` 2 supports only Python 3.7+.

Reply:
Replaced it, good point